### DISCUSSION TITLE
How to use @use and @include some.thing(...) in scss?

### DISCUSSION THREAD

**Discussion by sangdth** [vercel/next.js#11808] [Help]
I've tried to dig around Nextjs issue archive as well as Google a lot, but can't find the correct answer to make it works.

I just want to customize the `border-radius` from [Material's Shape module](https://github.com/material-components/material-components-web/tree/master/packages/mdc-shape), and according to the doc, all I have to do is call
```
@use '@material/shape'; // Warning: Unknown at rule @use
// then use it
@include shape.radius($radius);

```

But it will crash with error : 
<img width="650" alt="Screenshot 2020-04-10 at 17 48 16" src="https://user-images.githubusercontent.com/1083478/79001003-63345780-7b56-11ea-8739-ecf9974a57df.png">
```
SassError: Invalid CSS after "    @include shape": expected "}", was ".radius(0);"
        on line 5 of /Users/sangdth/Projects/saduak/front-end/assets/scss/global.scss
>>     @include shape.radius(0);
```

Please help. I'm hitting the head to the wall almost 3 days.

I'm using:
- Next: 9.3.3
- React 16.13.1
- Node: 12.14.1
- node-sass: 4.13.1
- sass: 1.26.3

---
Update:
OK, seems like I missed the [VERY BIG](https://carterbancroft.com/sass-and-webpack/) notice about Dart Sass:

![wow-best-ux](https://user-images.githubusercontent.com/1083478/79008383-f6c15480-7b65-11ea-903a-e6ffc8d7a5e7.gif)

**Comment by bcanedo4** [comment]
I was just as confused as you were. I find it random that users have to click on some side triangle to find that out.
